apps/src/redux/scriptSelectionRedux.js

```diff
@@ -50,6 +50,18 @@ export const getSelectedScriptFriendlyName = state => {
   return script ? script.name : null;
 };
 
+/* Get the description of a script(the unit or course name) */
+export const getSelectedScriptDescription = state => {
+  const scriptId = state.scriptSelection.scriptId;
+  if (!scriptId) {
+    return null;
+  }
+
+  const scripts = state.scriptSelection.validScripts;
+  const script = scripts.find(script => script.id === scriptId);
+  return script ? script.description : null;
+};
```

Comment:
Where is the description property coming from? I'd expect to see "description" here: https://github.com/code-dot-org/code-dot-org/blob/2364fe89fee1212406f491d41819af6d9869a5a8/apps/src/redux/scriptSelectionRedux.js#L56 if it were available on the validScript object.

Reply:
I got the description in script_constants.rb but I don't know that thats the right place for it and now its breaking some tests so I'm gonna take another look.